spring-integration-ip/src/test/java/org/springframework/integration/ip/dsl/IpIntegrationTests.java

```diff
@@ -164,6 +172,43 @@ public void testUdpInheritance() {
 		assertThat(udpMulticastOutboundChannelAdapterSpec2.get()).isInstanceOf(MulticastSendingMessageHandler.class);
 	}
 
+	@Test
+	public void testCloseStream() throws InterruptedException {
+		IntegrationFlow server = IntegrationFlows.from(Tcp.inboundGateway(Tcp.netServer(0)
+				.deserializer(new ByteArrayRawSerializer())))
+				.<byte[], String>transform(p -> "reply:" + new String(p).toUpperCase())
+				.bridge()
```

Comment:
I wonder why do we need bridge here...

Reply:
Good; I added the transformer later and should have removed the bridge.